pool/paymentmgr.go

```diff
@@ -877,35 +829,22 @@ func (pm *PaymentMgr) payDividends(ctx context.Context, height uint32, treasuryA
 
 	// Ensure the wallet is aware of all the coinbase outputs being
 	// spent by the payout transaction.
-	var maxSpendableHeight uint32
-	for _, pmtSet := range pmts {
-		spendableHeight := pmtSet[0].EstimatedMaturity + 1
-		if maxSpendableHeight < spendableHeight {
-			maxSpendableHeight = spendableHeight
-		}
-	}
-
-	var stopAfter uint32
-	switch {
-	case maxSpendableHeight > height:
-		stopAfter = maxSpendableHeight - height
-	default:
-		stopAfter = 1
-	}
-
-	err = pm.confirmCoinbases(pCtx, inputTxHashes, stopAfter)
+	err = pm.confirmCoinbases(pCtx, txB, inputTxHashes)
 	if err != nil {
-		atomic.AddUint32(&pm.failedTxConfs, 1)
-
-		// Track the transactions with inaccurate confirmation data.
-		pm.mtx.Lock()
-		for k, v := range inputTxHashes {
-			pm.txConfHashes[k] = v
+		if len(inputTxHashes) != 0 {
```

Comment:
This `if` seems unnecessary. How can an error occur if there are zero `inputTxHashes`?

Reply:
Yeah, that's no needed. Updating.